client/js/lounge.js

```diff
@@ -205,6 +207,7 @@ $(function() {
 			"action",
 		].indexOf(type) !== -1) {
 			switch (type) {
+			case "invite": data.msg.formattedAction = "invited " + data.msg.target + " to"; break;
```

Comment:
When testing, I was not able to see this displayed other than `invited you to #channel`. Wondering if that's a bug. What do I need to do to trigger this properly?

The steps I followed:

- user1 goes to #channel
- user1 types `/invite user2 #channel`
- user2 sees `user1 invited you to #channel` in the network window
- user2 clicks on `#channel` link as displayed in previous step
- Both users see `user2 has joined the channel` in #channel, no mention of something along the lines of `user1 invited user2 to this channel`

Am I misunderstanding something here?